### ISSUE TITLE
Repo page - remove 'view diff', change path to link to match stage page

### ISSUE THREAD

**Issue by FreHu** [larshp/abapGit#3268]
Unite behavior of the two pages

![image](https://user-images.githubusercontent.com/5097067/76700835-c77d0c00-66bb-11ea-99a3-327bddae8154.png)

by adding links and the button will become unnecessary:
![image](https://user-images.githubusercontent.com/5097067/76700828-b7fdc300-66bb-11ea-873e-c63646a6e15b.png)

Unite column names in both screens to `Path (click to view diff)`

**Comment by sbcgua** [comment]
Addition from side: clicking path or object should display diff

Technical consideration: I would probably refactor how the parts of the page are rendered now. Imho I'd make the header + footer kind of high order component and the content would be a child component. Both would be separate view classes and the pages then become kind of controllers. Well, if this was clear :) Maybe will have a look on this

**Comment by FreHu** [comment]
You diff per file, but for example a class xml file does not have a corresponding object part to navigate to. So I think it makes more sense to have adt link be the object name, although it would prevent navigation to object parts when it is possible.

**Comment by sbcgua** [comment]
I'd make navigation completely separate. E.g. I hide files. So will there be no diffs ? Both should lead to diff page imho. In the same way as the `view diff` now (just the link should be on the text). The navigation to the object - imho make it as a hover link near the object name. Kind of 

![image](https://user-images.githubusercontent.com/15635498/77234266-a03fa680-6bb5-11ea-8e09-e01fee36cebd.png)
 but larger, probably with text ("open object")

**Comment by sbcgua** [comment]
or maybe open object on table line click - this should be convenient but maybe not obvious enough ...

**Comment by FreHu** [comment]
#5323 makes further adjustments, but the pages are still not consistent at all. This issue was dead for some time, maybe it's time to refresh opinions.

- stage does not show objects, just files
- status should be left/right?
- link to diff as icon/click on path?

We could get away with putting the status on the left and having object type for objects, status for files. But shouldn't forget about the folder view...
![image](https://user-images.githubusercontent.com/5097067/152978272-97e623c8-57c4-4aff-a345-03896bf352b5.png)

**Comment by sbcgua** [comment]
Imho:

- status should be on the left. Because on large screens it is difficult to track correspondent items
- icon is not needed. Because no additional value
- diff should be entered by clicking the object (or file). Because it is more intuitive and more frequent operation then navigating to the object. Also for the same reason as status
- I'd say default display more should be the objects, not the files. Does anyone check each file diff separately ?
- the stage should be probably merged with repo page. Or (maybe better in terms of code structure) be based on the same table component, so that repo and stage look as similar as possible. Merge into one page might be not ideal sinse the stage page has some own logic and if we add checkboxes it would look cluttered
- folder view ... maybe not needed. As far as I remember I created it in the first place, but not using it. I'd better add a filtering feature at the top

**Comment by sbcgua** [comment]
> it would certainly simplify things if it wasn't there, maybe package could instead be added as an optional column? Then sorting by package gives you the (expanded) folders view

Good idea, but if the column is optional. It would go well with the filtering (which should take this column into account too)

**Comment by kjetil-kilhavn** [comment]
> 



> The navigation to the object - imho make it as a hover link near the object name. Kind of
> 
> ![image](https://user-images.githubusercontent.com/15635498/77234266-a03fa680-6bb5-11ea-8e09-e01fee36cebd.png) but larger, probably with text ("open object")

Why not use the already existing icon at the left to navigate to the object? If that is too small as click target then both the icon and the type could be used.
